spec/factories/valid_draft_factories.rb

```diff
@@ -815,7 +819,7 @@
           'Eon' => 'test 3 Eon text 1'
         }]
       }],
-      'ISOTopicCategories' => %w(farming climatologyMeteorologyAtmosphere health)
+      'ISOTopicCategories' => %w[farming climatologyMeteorologyAtmosphere health]
```

Comment:
%w[]!? really.. thats odd. Now we'll have a bunch of warning for the previous version hah